modules/swagger-codegen/src/main/resources/Java/StringUtil.mustache

```diff
@@ -48,4 +48,12 @@ public class StringUtil {
     if (o == null) return "null";
     return o.toString().replace("\n", "\n    ");
   }
+
+  public static boolean isNotBlank(String s){
+    if(s != null && !s.trim().equals("")){
```

Comment:
Also, while precedence makes this pretty clear, I'd still use parens around the predicates.